bin/fuel-core/src/cli/run/p2p.rs

```diff
@@ -66,6 +66,10 @@ pub struct P2PArgs {
     #[clap(long = "max-headers-per-request", default_value = "100", env)]
     pub max_headers_per_request: u32,
 
+    /// Max number of blocks in a transaction request response
+    #[clap(long = "max-blocks-per-tx-request", default_value = "100", env)]
+    pub max_blocks_per_tx_request: u32,
```

Comment:
I think we can reuse `max_headers_per_request` to reduce number of arguments

Reply:
addressed in https://github.com/FuelLabs/fuel-core/pull/2112/commits/06262b9b8c210ac8b485d96050f52a54d96d18f4